jsapp/js/components/modalForms/projectSettings.es6

```diff
@@ -750,6 +781,36 @@ class ProjectSettings extends React.Component {
               <iframe src={this.props.iframeUrl} />
             </bem.FormView__cell>
           }
+
+          {this.props.context === PROJECT_SETTINGS_CONTEXTS.EXISTING &&
+            <bem.FormModal__item>
+              <bem.FormModal__item m='inline'>
+                <button
+                  type='button'
+                  className='mdl-button mdl-button--colored mdl-button--danger mdl-button--raised'
```

Comment:
I've changed the archive button to orange (because it is in a way more serious than normal button), and unarchive to blue (to match some other place that has unarchive button).

Reply:
<img width="338" alt="screenshot 2018-12-10 at 12 14 33" src="https://user-images.githubusercontent.com/2521888/49729247-3546b680-fc75-11e8-85dc-888f16d0035f.png">
![Uploading Screenshot 2018-12-10 at 12.14.43.png…]()